### PR TITLE
adjust composer.json to be called dabilite now

### PR DESCRIPTION
**Pull request by thyseus**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "dektrium/yii2-user",
+    "name": "dabilite/yii2-user",
     "description": "Flexible user registration and authentication module for Yii2",
```
Comment (SamMousa) [inline composer.json:3]: Mention that this is a fork?

Comment (thyseus) [reply composer.json:3]: Disagree. Merging back into dektrium (hopefully soon) will be as easy as replacing this single string again.

Comment (SamMousa) [reply composer.json:3]: How will we prevent anyone else from just adding this to packagist btw?

Comment (thyseus) [reply composer.json:3]: Wouldn´t hurt imo.

Comment (thyseus) [reply composer.json:3]: https://snippets.khromov.se/composer-use-your-own-fork-for-a-package/ works - i close this issue.

**Pull request closed without merging**